solidity/test/maintainer/MaintainerProxy.test.ts

```diff
@@ -0,0 +1,3434 @@
+        })
+
+        context("when the transaction has multiple inputs", () => {
+          context(
+            "when the input is marked as correctly spent in the Bridge",
+            () => {
+              const data = witnessSignMultipleInputTx
+              let tx: Promise<ContractTransaction>
+              let initThirdPartyBalance: BigNumber
+
+              before(async () => {
+                await createSnapshot()
+
+                await bridge.setWallet(walletPublicKeyHash, {
+                  ecdsaWalletID: ethers.constants.HashZero,
+                  mainUtxoHash: ethers.constants.HashZero,
+                  pendingRedemptionsValue: 0,
+                  createdAt: await lastBlockTime(),
+                  movingFundsRequestedAt: 0,
+                  closingStartedAt: 0,
+                  pendingMovedFundsSweepRequestsCount: 0,
+                  state: walletState.Live,
+                  movingFundsTargetWalletsCommitmentHash:
+                    ethers.constants.HashZero,
+                })
+                await bridge.setSweptDeposits(data.deposits)
+                await bridge.setSpentMainUtxos(data.spentMainUtxos)
+
+                await bridge
+                  .connect(thirdParty)
+                  .submitFraudChallenge(
+                    walletPublicKey,
+                    data.preimageSha256,
+                    data.signature,
+                    {
+                      value: fraudChallengeDepositAmount,
+                    }
+                  )
+
+                initThirdPartyBalance = await provider.getBalance(
+                  authorizedMaintainer.address
+                )
+
+                tx = maintainerProxy
+                  .connect(authorizedMaintainer)
+                  .defeatFraudChallenge(
+                    walletPublicKey,
+                    data.preimage,
+                    data.witness
+                  )
+              })
+
+              after(async () => {
+                await restoreSnapshot()
+              })
+
+              it("should not revert", async () => {
+                await expect(tx).not.to.be.reverted
+              })
+
+              it("should refund ETH", async () => {
+                const postWalletRegistryBalance = await provider.getBalance(
+                  authorizedMaintainer.address
+                )
+                const diff = postWalletRegistryBalance.sub(
+                  initThirdPartyBalance
+                )
+
+                expect(diff).to.be.gt(0)
+                expect(diff).to.be.lt(
+                  ethers.utils.parseUnits("2000000", "gwei") // 0,002 ETH
+                )
+              })
+            }
+          )
+        })
+      })
+    })
+  })
+
+  describe("submitMovingFundsProof", () => {
+    context("when there is only one input", () => {
+      before(async () => {
+        await createSnapshot()
+      })
+
+      after(async () => {
+        walletRegistry.requestNewWallet.reset()
```

Comment:
Why do we need this line?

Reply:
We don't. Removed in https://github.com/keep-network/tbtc-v2/pull/209/commits/000f0dc96c572d263b1035a5dd6cad85f98a4a50.